packages/@react-aria/tabs/src/useTabPanel.ts

```diff
@@ -53,20 +53,17 @@ export function useTabPanel<T>(props: AriaTabPanelProps, state: TabListState<T>,
     }
   }, [ref]);
 
-  props = {
-    id: props.id,
-    'aria-describedby': props['aria-describedby'],
-    'aria-details': props['aria-details'],
-    'aria-label': props['aria-label'],
-    'aria-labelledby': props['aria-labelledby']
-  };
+  const id = generateId(state, state?.selectedKey, 'tabpanel');
+  const tabPanelProps = useLabels({...props, id});
 
   return {
-    tabPanelProps: mergeProps(props, {
-      id: generateId(state, state?.selectedKey, 'tabpanel'),
+    tabPanelProps: mergeProps(tabPanelProps, {
+      id,
       'aria-labelledby': generateId(state, state?.selectedKey, 'tab'),
       tabIndex,
-      role: 'tabpanel'
+      role: 'tabpanel',
+      'aria-describedby': props['aria-describedby'],
+      'aria-details': props['aria-details']
```

Comment:
so the returned object of useLabels is 
```
  return {
    id,
    'aria-label': label,
    'aria-labelledby': labelledBy
  };
```
so we no longer need to include those three in the second object to mergeProps

```
    tabPanelProps: mergeProps(tabPanelProps, {
      tabIndex,
      role: 'tabpanel',
      'aria-describedby': props['aria-describedby'],
      'aria-details': props['aria-details']
    });
```

Reply:
Fixed.

I need to overwrite the `aria-labelledby`, but I could do it in the `useLabels`.